src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AutoInsert/OnAutoInsertEndpoint.cs

```diff
@@ -115,7 +115,7 @@ public async Task<OnAutoInsertResponse> Handle(OnAutoInsertParams request, Cance
             var uri = request.TextDocument.Uri;
             var position = request.Position;
 
-            using (var formattingContext = FormattingContext.Create(uri, document, codeDocument, request.Options, _workspaceFactory, isFormatOnType: false, automaticallyAddUsings: false))
+            using (var formattingContext = FormattingContext.Create(uri, document, codeDocument, request.Options, _workspaceFactory, isFormatOnType: false, automaticallyAddUsings: false, hostDocumentIndex: 0, triggerCharacter: '\0'))
```

Comment:
Could we put the trigger character as a constant?

Reply:
I've separated out things to two `Create` methods so the intent is clear, and nobody needs to pass in `'\0'`